Added svmon support

Agupane opened the pull request.

Agupane committed:
Added svmon support

kouril commented on bin/pakiti-client:
```diff
@@ -238,6 +238,26 @@ sub find_packages ($) {
+	
+    #The user wants to send svmon data
+    $cmd = $Option{"svmonreport"};
+    if ($cmd) {
```
If you change these two lines to:
  if ($Option{"svmonreport"}) {
we'll gain quite some readability (without misusing the $cmd variable ;-)

kouril commented on bin/pakiti-client:
```diff
@@ -452,6 +472,8 @@ sub init () {
         "rndsleep" => "|r=i",
         "site"     => "=s",
+        "svmonreport" => "|s",
+        "svmon" => "=s",
```
a bit nitpicking, but please align the => chars with those below (or above the longer svmonreport option)

kouril commented on bin/pakiti-client:
```diff
@@ -672,6 +694,14 @@ set the path of the C<rpm> command to use
 
+=item B<--svmonreport>, B<-s>
+
+show only the report with svmon data
```
please refer to EUDAT somehow so that's clear what the SVMON is

Agupane committed:
Added SVMON Support in pakiti.
SVMON collects the information on software versions of EUDAT services and
their components installed in EUDAT CDI.

The framework allows the monitoring of the following information:
-Operating system flavor and version of the endpoint.
-EUDAT service component software versions.
-Any service software version installed on the endpoint.
-History of changes performed for the given service.

Agupane committed:
Added SVMON Support in pakiti.
SVMON collects the information on software versions of EUDAT services and
their components installed in EUDAT CDI.

The framework allows the monitoring of the following information:
-Operating system flavor and version of the endpoint.
-EUDAT service component software versions.
-Any service software version installed on the endpoint.
-History of changes performed for the given service.

Agupane committed:
Merge branch 'svmon' of https://github.com/Agupane/pakiti-client into svmon
